contracts/contracts/strategies/ConvexGeneralizedMetaStrategy.sol

```diff
@@ -140,12 +140,15 @@ contract ConvexGeneralizedMetaStrategy is BaseConvexMetaStrategy {
 
         uint256 burnAmount = metapoolLPToken.balanceOf(address(this));
         if (burnAmount > 0) {
+            uint256 burnDollarAmount = burnAmount.mulTruncate(metapool.get_virtual_price());
```

Comment:
What happens when the price is either too high or too low? Should we be worried about burning too much or too less OUSD?

Wondering if a min/max slippage tolerance should be added here too

Reply:
This strategy is for non OUSD metapools. Here we burn metapoolLP tokens to receive only one of the coins in the pool - 3crvLP. That is done below with the `metapool.remove_liquidity_one_coin`  call